changelogs/fragments/filetree_create_issue130.yaml

```diff
@@ -0,0 +1,4 @@
+---
+breaking_changes:
```

Comment:
why is this a breaking change? if you are just adding the ability to export objects how does it break prior versions?

Reply:
I'm not sure what I was thinking about :-D... fixing this.